src/FixedPointDecimals.jl

```diff
@@ -38,6 +38,31 @@ import Base: reinterpret, zero, one, abs, sign, ==, <, <=, +, -, /, *, div, rem,
 
 include("fldmod_by_const.jl")
 
+# =========================================================
+# Set Int128 to widen to Int256 within FixedPointDecimals to avoid allocations from BigInt.
+# ---------------------------------------
+
+import BitIntegers
+
+# BitIntegers is missing unsigned(::Type{Int256})
+Base.unsigned(x::T) where T<:BitIntegers.XBS = reinterpret(unsigned(T), x)
```

Comment:
What is XBS?  eXtra Big Stick?

Reply:
Haha i guess it's "eXtended Bit (integers) Signed". Blech!

But also, i just switched it to `XBI`, which is "eXtended Bit Integers", which is what I settled on in the PR i sent to that package: https://github.com/rfourquet/BitIntegers.jl/pull/2

It's basically the super type for all the new integer types that package added.

I clarified that in the comments, and also added a TODO comment to delete these lines once the PRs on the base package are merged. Thanks! :)